XMonad/Actions/Search.hs

```diff
@@ -300,11 +290,9 @@ hackage       = searchEngine "hackage"       "http://hackage.haskell.org/package
 hoogle        = searchEngine "hoogle"        "http://hoogle.haskell.org/?hoogle="
 images        = searchEngine "images"        "http://images.google.fr/images?q="
 imdb          = searchEngine "imdb"          "http://www.imdb.com/find?s=all&q="
-isohunt       = searchEngine "isohunt"       "http://isohunt.com/torrents/?ihq="
 lucky         = searchEngine "lucky"         "http://www.google.com/search?btnI&q="
 maps          = searchEngine "maps"          "http://maps.google.com/maps?q="
 mathworld     = searchEngine "mathworld"     "http://mathworld.wolfram.com/search/?query="
-openstreetmap = searchEngine "openstreetmap" "http://gazetteer.openstreetmap.org/namefinder/?find="
```

Comment:
I'm not sure about the other two, but name finder was effectively replaced by [nominatim](https://nominatim.openstreetmap.org/ui/search.html), which offers much of the same functionality.  It would seem prudent to just replace the URLs, instead of removing the functionality entirely

Reply:
Yes, you are right. I have checked how those URLs changed and there is a replacement for those. I updated them accordingly. 

I also added Github search engine as it may be a good addition. 

I am not sure about isohunt; although there is a couple of pages with that URLs their status may not be legal and/or their URLs might change a lot.

Reply:
Yeah, probably fine leaving isoHunt out of there; not like anyone has complained about it not working up to this point :)